manifests/a/Azul/Zulu/14/JDK/14.29.23/Azul.Zulu.14.JDK.locale.zh-CN.yaml

```
@@ -0,0 +1,34 @@
+# Created with YamlCreate.ps1 v2.4.1 Dumplings Mod $debug=QUSU.CRLF.7-4-2.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.locale.1.6.0.schema.json
+
+PackageIdentifier: Azul.Zulu.14.JDK
+PackageVersion: 14.29.23

```

```suggestion
PackageVersion: "14.29"
```
Please update file path to match this change.